server/models/mv/bilibili/provider.js

```diff
@@ -2,15 +2,41 @@ import fetch from "node-fetch";
 import Entry, { NAME } from "./entry";
 
 class Provider {
+  cookie;
+
+  constructor() {
+    // TODO: It's async but should work most time.
+    this._initCookie();
```

Comment:
I think there will be racing between the async init and init on searching.
Let us remove it and only initialize cookies on the first searching.

Reply:
Ok, that's reasonable.

Reply:
Done